### ISSUE TITLE
Implementing Striving for Simplicity: The All Convolutional Net in Tensorflow: results on the test set lower than expected

### ISSUE THREAD

**Issue by jeandut** [tensorflow/tensorflow#1271]
I posted [this issue with my code](http://stackoverflow.com/questions/35339636/erratic-training-for-all-cnn-on-cifar-10) on stackoverflow a while ago but it has not received any answer or comment.
The preprocessing I used is in the dataset class (that I adapted from the Deep MNIST for experts tutorial) and is exactly the same as in the neon/pylearn2 implementation: calculating the mean and Wzca matrix on training set and use it to whiten the data on the training and test set followed by a global contrast normalization step with Goodfellow scale factor of 55.    
The only difference of my implementation according to me (I would like to be mistaken) with the paper is the use of `tf.reduce_mean` instead of `tf.reduce_sum` (which led me to exploding ReLU grad).
So I divide the weight decay by a factor batch_size to keep the change.    
With this I got up to 85.something% on the test set instead of the 91% claimed by the authors.  
Do you see something wrong immediately ? For instance I feel the weight decay part of the cross entropy is ugly but I could not find any better way of doing that. Do you have some ideas ?  
If you think that this is not an appropriate place for my question tell me I will remove it but I do not know where to look.

**Comment by Dringite** [comment]
This is a place to post bugs with the software and not problems with particular implementations.
You yourself stated that you did not follow the paper properly, why are you surprised at getting different results?

Also the weights may have been initialized poorly (or due to a bad training rate) leading to many dead neurons (ReLU's that will never respond with anything other than 0 to the entire training data manifold), no one here will spend time debugging your model for you. If you can't do it then the best course of action is to learn.

**Comment by vrv** [comment]
As the now-deleted-comment suggested, this type of detailed debugging is likely out of scope for GitHub -- you may get better help on StackOverflow or such.
